shell/environment.go

```diff
@@ -50,29 +160,6 @@ func (e *Environment) Exists(key string) bool {
 
 // Sets a key in the environment
 func (e *Environment) Set(key string, value string) string {
-	// Trim the values
```

Comment:
I remove all this code because it's all wrong. We shouldn't be doing any of that stuff with environment variable values.